core/cmd/shell_local.go

```diff
@@ -73,11 +77,15 @@ func initLocalSubCmds(s *Shell, safe bool) []cli.Command {
 			},
 			Usage:  "Run the Chainlink node",
 			Action: s.RunNode,
+			Before: s.InitStartComponents,
+			After:  s.CleanupStartComponents,
```

Comment:
Why don't we align these names?
```suggestion
			Before: s.BeforeNode,
			After:  s.AfterNode,
```

Reply:
Done